docker/stan-wasm-server/logic/compilation.py

```diff
@@ -38,60 +37,46 @@ def copy_compilation_outputs(*, model_dir: Path, job_dir: Path):
     for (src, dest) in todos:
         copy2(src, dest)
```

Comment:
Is this section still needed? The note says to remove it, but not sure where we landed.

Reply:
Yeah, dead code